src/java/htsjdk/samtools/util/Tuple.java

```diff
@@ -13,4 +13,26 @@ public Tuple(final A a, final B b) {
         this.a = a;
         this.b = b;
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
These will all break if A or B is null.  Recommend using the autogenerated intellij equals + hashcode unless you have a good reason not to